src/lib/webpack/dependency-install-loader.js

```diff
@@ -30,19 +31,15 @@ const CACHE = {};
 
 function isInstalled(dep) {
 	return CACHE[dep] || (CACHE[dep] = new Promise( resolve => {
-		fs.stat(resolve('node_modules', dep), err => {
+		fs.stat(path.resolve('node_modules', dep), err => {
 			resolve(!err);
 		});
 	}));
 }
 
 function installDeps(deps, save) {
 	process.stdout.write(`\nInstalling ${deps.join(' ')}..`);
-	return spawn('npm', [
-		'install', save && (save==='dev' ? '--save-dev' : '--save'),
-		'--prefix', process.cwd(),
-		...deps
-	].filter(Boolean)).then( () => {
+	return install(false, process.cwd(), deps, save).then( () => {
```

Comment:
Would it make sense to allow yarn here if it's installed?  We could check for a `yarn.lock` as an indication of whether the current repo is set up with yarn.